en/documents.html

```diff
@@ -164,6 +164,13 @@ <h2 id="document-expiry">Document expiry</h2>
     }
 }
 </pre>
+<p>
+  When <code>garbage-collection="true"</code>, Vespa iterates through the document space to purge expired documents.
+  It is not possible to observe where in the cycle the process it,
+  but one can configure a pause between cycles using
+  <a href="reference/services-content.html#documents.selection">garbage-collection-interval</a>.
```

Comment:
\*Pushes glasses up to nose\*

_Technically_ 🤓 the GC is always running in the background when enabled. Vespa distributes future GC deadlines for all buckets pseudo-randomly within the configured time interval (it chunks the wall-clock timeline into discrete, non-overlapping intervals of this size). The interval is therefore not a pause as per se, but the minimum time between each time a particular bucket (and therefore a particular document, assuming it stays in the same bucket) is considered for GC.

But it's definitely the case that you cannot know for sure when a document will be GC'd.